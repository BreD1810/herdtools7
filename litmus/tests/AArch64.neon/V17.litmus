AArch64 V17
(* Tests MOV element *)
{
0:X0=x;0:X1=y;0:X2=z;
}

 P0                     ;
 MOVI V0.8B,#1          ;
 MOV V1.B[0], V0.B[0]   ;
 MOV V1.B[1], V0.B[7]   ;
 ST1 {V1.B}[0], [X0]    ;
 ST1 {V1.B}[1], [X1]    ;
 ST1 {V1.B}[2], [X2]    ;
 forall (x=1 /\ y=1 /\ z=0)
