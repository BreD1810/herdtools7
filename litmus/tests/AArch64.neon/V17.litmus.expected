Test V17 Required
Histogram (1 states)
1000000:>x=1; y=1; z=0;
Ok

Witnesses
Positive: 1000000, Negative: 0
Condition forall (x=1 /\ y=1 /\ z=0) is validated
Hash=8150239dd3282ba7fe22da0c7679d825
Observation V17 Always 1000000 0
